### PR TITLE
1195 bulk insert accepts both address and addressLine

### PR DESCRIPTION
**Pull request by leite08**
Ref: metriport/metriport-internal#1195

### Dependencies

none

### Description

Bulk insert accepts both address and addressLine

### Release Plan

- nothing special

### CODE REVIEW

File: packages/api/src/routes/helpers/request-analytics.ts [hunk 1]
```diff
@@ -34,8 +34,8 @@ export const analyzeRoute = ({
   const headerCxId = getCxIdFromHeaders(req);
   const cxId = reqCxId ?? headerCxId;
 
-  const isMedical = medicalRoutes.some(route => req.baseUrl.includes(route));
-  const isDevices = devicesRoutes.some(route => req.baseUrl.includes(route));
+  const isMedical = medicalRoutes.some(route => url.includes(route));
+  const isDevices = devicesRoutes.some(route => url.includes(route));
```
Comment (leite08) [inline packages/api/src/routes/helpers/request-analytics.ts:38]: https://metriport.slack.com/archives/C04T256DQPQ/p1699488089496119

File: packages/utils/src/bulk-insert-patients.ts [hunk 2]
```diff
@@ -104,7 +104,8 @@ function toTitleCase(str: string): string {
     .trim();
 }
 
-function normalizeGender(gender: string): "M" | "F" {
+function normalizeGender(gender: string | undefined): "M" | "F" {
+  if (gender == undefined) throw new Error(`Missing gender`);
```
Comment (Goncharo) [inline packages/utils/src/bulk-insert-patients.ts:108]: nit: === ?

File: packages/utils/src/bulk-insert-patients.ts [hunk 3]
```diff
@@ -114,13 +115,15 @@ function normalizeGender(gender: string): "M" | "F" {
   throw new Error(`Invalid gender ${gender}`);
 }
 
-function normalizeName(name: string): string {
+function normalizeName(name: string | undefined, propName: string): string {
+  if (name == undefined) throw new Error(`Missing ` + propName);
```
Comment (Goncharo) [inline packages/utils/src/bulk-insert-patients.ts:119]: Ditto

File: packages/utils/src/bulk-insert-patients.ts [hunk 4]
```diff
@@ -114,13 +115,15 @@ function normalizeGender(gender: string): "M" | "F" {
...
   return toTitleCase(name);
 }
 
 const phoneRegex = /^\+?1?\d{10}$/;
 
-function normalizePhone(phone: string): string | undefined {
+function normalizePhone(phone: string | undefined): string | undefined {
+  if (phone == undefined) throw new Error(`Missing phone`);
```
Comment (Goncharo) [inline packages/utils/src/bulk-insert-patients.ts:126]: Ditto

**Review by Goncharo — COMMENTED** [review]
Some nits about all the places we're doing == checks, also, do we want to add support for phone2/email2?

File: packages/utils/src/bulk-insert-patients.ts (hunk 2 above)
Comment (leite08) [reply packages/utils/src/bulk-insert-patients.ts:108]: Intentionally `==` so it catches both `undefined` and `null`

**Comment by leite08** [comment]
> do we want to add support for phone2/email2

@Goncharo this is what we discussed [here](https://metriport.slack.com/archives/C0616FCPAKZ/p1699490035005159?thread_ts=1699473593.171269&cid=C0616FCPAKZ), do you think we want to capture those but not use them?

**Review by Goncharo — APPROVED** [review]
🚢

**Pull request merged**